src/data/matrix/basic.lean

```diff
@@ -934,6 +936,28 @@ lemma minor_mul_equiv [semiring α] {p q : Type*} [fintype p] [fintype q]
   (M ⬝ N).minor e₁ e₃ = (M.minor e₁ e₂) ⬝ (N.minor e₂ e₃) :=
 minor_mul M N e₁ e₂ e₃ e₂.bijective
 
+lemma mul_minor_one [semiring α] [decidable_eq o] (e₁ : n ≃ o) (e₂ : l → o) (M : matrix m n α) :
+  M.mul ((1 : matrix o o α).minor e₁ e₂) = minor M id (e₁.symm ∘ e₂) :=
+begin
+  let A := M.minor id e₁.symm,
+  have : M = A.minor id e₁,
+  { simp only [minor_minor, function.comp.right_id, minor_id_id, equiv.symm_comp_self], },
+  rw [this, ←minor_mul_equiv],
+  simp only [matrix.mul_one, minor_minor, function.comp.right_id, minor_id_id,
+    equiv.symm_comp_self],
+end
+
+lemma mul_one_minor [semiring α] [decidable_eq o] (e₁ : l → o) (e₂ : m ≃ o) (M : matrix m n α) :
```

Comment:
Here it is.

Reply:
```suggestion
lemma one_minor_mul [semiring α] [decidable_eq o] (e₁ : l → o) (e₂ : m ≃ o) (M : matrix m n α) :
```
seems more consistent with `one_mul`

Reply:
Thank you for your help! I committed all your recent changes, but I was thinking about the one concerning `one_minor_mul`. If the idea is that `mul` is in the middle of two factors, which I find consistent with `one_mul` (the second factor being absent, since it is universal), the first lemma should be `minor_mul_one` and the second ` one_mul_minor`. What do you think? After this, I will send this to `bors`.

Reply:
The one and minor are both on the same side of the mul: `one_mul_minor` would imply `1 ⬝ M.minor _ _`, not the `1.minor _ _ ⬝ M` that we're naming here

Reply:
Oh sure, you're right. Ok, I'll take your suggestion.